CMakeLists.txt

```diff
@@ -47,6 +46,12 @@ if (UNIX)
 
 endif ()
 
+if (CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
+  add_compile_options("-std=c++11")
+elseif (CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
Looks very suspicious that both conditions are equal. Also if OSX is considered to be UNIX by CMake, I would rather keep it in the UNIX block above for readability.

Reply:
Bad copy/paste! The goal was to move those linker flags for Linux only. I will fix it soon.